src/xcode/ENA/ENA/Source/Services/__tests__/Mocks/MockTestStore.swift

```diff
@@ -83,12 +83,20 @@ final class MockTestStore: Store, PPAnalyticsData {
 	
 	var selectedLocalStatisticsRegions: [LocalStatisticsRegion] = []
 
+	// MARK: - SRS Providing
+
+	var ppacApiTokenSRS: TimestampedToken?
```

Comment:
```suggestion
	var ppacApiTokenSrs: TimestampedToken?
```

Reply:
Why should it be Srs not SRS, we are writing PPAC not Ppac for example